packages/shared/src/helpers/mapToGuidelineDesignTokens.ts

```diff
@@ -22,13 +22,16 @@ const TokenNameMapper: Record<string, DesignTokenName> = {
 const transformDesignTokens = (dataToTransform: DesignTokenProperties) => {
     const cssStyles: TokenValues = {};
 
-    for (const [key, value] of Object.entries(dataToTransform)) {
-        if (typeof value === 'object') {
-            transformObjectValues(key, cssStyles, value);
-        } else {
-            transformStringValues(key, cssStyles, value);
+    if (dataToTransform) {
```

Comment:
you could set a default value for the parameter instead

Reply:
Not possible to have a default value, because it could be any css property. Changed it to return `null` now.